spec/features/admin_deletes_comments_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "admin can delete comments", %{
+  As a admin
+  I want to delete comments
+  So that I can keep content updated
+
+  Acceptance Criteria
+  * Admin must be able to delete comments from admin panel
+  } do
+
+  scenario "admin can delete comments" do
+    comments = FactoryGirl.create_list(:comment, 10)
+
+    admin = FactoryGirl.create(:user, admin: true)
+
+    sign_in(admin)
+
+    visit admin_comments_path
+
+    first(".comment").click_link("Delete comment")
+
+    expect(page).to have_content "Comment Deleted"
+    expect(comments.count).to eq 10
```

Comment:
If we are creating 10 comments and deleting one, should we be expecting comments.count to equal 9?